api/src/test/java/org/itsallcode/openfasttrace/api/core/TestLinkedSpecificationItem.java

```diff
@@ -36,10 +36,15 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
 
 // [utest->dsn~linked-specification-item~1]
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
```

Comment:
Fix unnecessary stubbings instead of using lenient mode, please.

Reply:
This will cause a lot of code duplication. But i will do it if you prefer ;)

Reply:
Will be fixed in #256